README.md

```diff
@@ -1,9 +1,53 @@
+[![Build Examples](https://github.com/ARMmbed/mbed-os-example-ble-fota/actions/workflows/build-examples.yml/badge.svg?branch=main)](https://github.com/ARMmbed/mbed-os-example-ble-fota/actions/workflows/build-examples.yml)
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 
-> __Note__: The contents of this repository are currently being migrated from the `fota-service-example` branch in the [mbed-os-example-ble](https://github.com/ARMmbed/mbed-os-example-ble) repository. 
+This repository houses example applications built for the Arm® Mbed™ OS platform that demonstrate the usage and abilities of the Bluetooth® Low Energy (BLE) firmware-over-the air (FOTA) service. The service specification and sources can be found [here](https://github.com/ARMmbed/mbed-os-experimental-ble-services/tree/fota-service-github-ci/services/FOTA). As defined in the specification, the FOTA service facilitates the transfer of firmware updates over BLE.
+
+The examples come with _batteries included!_ They're bundled with an automated build tool, "fota.sh", that eases the build process for the end-user. Please refer to the sections below for more information.
+
+## Pre-requisites
+Currently, the only supported target board for the examples is the [`NRF52840_DK`](https://os.mbed.com/platforms/Nordic-nRF52840-DK/). Development work to support [`DISCO_L475VG_IOT01A`](https://os.mbed.com/platforms/ST-Discovery-L475E-IOT01A/) will commence soon. The [BLE documentation](https://os.mbed.com/docs/mbed-os/v6.12/apis/ble.html) describes the BLE APIs available on Mbed™ OS; going through this documentation isn't strictly essential to run the examples, but is helpful in understanding the sources.
+
+## Build Tool
+The `fota.sh` cli tool aids in setup and build of the examples in this repository. The tool takes in as arguments the example, target board, mount point of the target board, and toolchain. Please run `./scripts/fota.sh --help` for usage instructions.
+
+> **Note**: If the mount point is not provided (it could be the case that an end-user is trying to build without the board connected), then the target binary is not flashed. This is especially useful for building the examples with a CI workflow.
 >
-> To run the MCUBoot example, please refer to this [link](https://github.com/ARMmbed/mbed-os-example-ble/tree/fota-service-example/FOTA/MCUboot), and to run the Mock example, please refer to this [link](https://github.com/ARMmbed/mbed-os-example-ble/tree/fota-service-example/FOTA/Mock). The FOTA service specification overview can be found [here](https://github.com/ARMmbed/mbed-os-experimental-ble-services/blob/fota-service-github-ci/services/FOTA/docs/README.md).
+> In the case of the MCUboot example, the _"factory firmware"_ is saved and would require manual transfer when the board is indeed connected. In either case, the demonstration step would be the only part that requires manual intervention from the user.
```

Comment:
What is factory firmware?

Reply:
Factory firmware,`merged.hex`, is the result of merging the bootloader and the initial signed application binaries. This happens in the 13th step of the build function in `scripts/mcuboot.sh`.